mappings/net/minecraft/resource/ServerResourceManager.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_5350 net/minecraft/resource/ServerResourceManager
+	FIELD field_25335 resourceManager Lnet/minecraft/class_3296;
+	FIELD field_25336 commandManager Lnet/minecraft/class_2170;
+	FIELD field_25337 recipeManager Lnet/minecraft/class_1863;
+	FIELD field_25338 registryTagManager Lnet/minecraft/class_3505;
+	FIELD field_25339 lootConditionManager Lnet/minecraft/class_4567;
+	FIELD field_25340 lootManager Lnet/minecraft/class_60;
+	FIELD field_25341 serverAdvancementLoader Lnet/minecraft/class_2989;
+	FIELD field_25342 functionManager Lnet/minecraft/class_5349;
+	METHOD <init> (ZI)V
+		ARG 1 isDedicatedServer
+	METHOD method_29465 functionManager ()Lnet/minecraft/class_5349;
+	METHOD method_29466 reload (Ljava/util/List;ZILjava/util/concurrent/Executor;Ljava/util/concurrent/Executor;)Ljava/util/concurrent/CompletableFuture;
+	METHOD method_29468 lootConditionManager ()Lnet/minecraft/class_4567;
+	METHOD method_29469 lootManager ()Lnet/minecraft/class_60;
+	METHOD method_29470 registryTagManager ()Lnet/minecraft/class_3505;
+	METHOD method_29471 recipeManager ()Lnet/minecraft/class_1863;
+	METHOD method_29472 commandManager ()Lnet/minecraft/class_2170;
+	METHOD method_29473 serverAdvancementLoader ()Lnet/minecraft/class_2989;
+	METHOD method_29474 resourceManager ()Lnet/minecraft/class_3300;
```

Comment:
Should these have `get` prefixes?

Reply:
Yes, I kinda forgot about that everwhere. Thats what I get for moving around projects with diffrent conventions lol